src/EnergyPlus/DaylightingManager.cc

```diff
@@ -731,53 +727,34 @@ namespace DaylightingManager {
-                                      Surface(IWin).Name,
-                                      DaylFac1,
-                                      DaylFac2);
-
-                                // Overcast sky
-                                DaylFac1 = ZoneDaylight(ZoneNum).DaylIllFacSky(12, 1, 4, 1, loop);
-                                DaylFac2 = 0.0;
-                                if (ZoneDaylight(ZoneNum).TotalDaylRefPoints > 1) DaylFac2 = ZoneDaylight(ZoneNum).DaylIllFacSky(12, 1, 4, 2, loop);
-                                print(outputFiles.eio,
-                                      " Overcast Sky Daylight Factors,{},{},{},{:.4R},{:.4R}\n",
-                                      CurMnDy,
-                                      Zone(ZoneNum).Name,
-                                      Surface(IWin).Name,
-                                      DaylFac1,
-                                      DaylFac2);
+                                // Output for each reference point, for each sky. Group by sky type first
+                                for (const SkyType& skyType: {SkyType::Clear, SkyType::ClearTurbid, SkyType::Intermediate, SkyType::Overcast}) {
+                                    std::string skyTypeString;
+                                    if (skyType == SkyType::Clear) {
+                                        skyTypeString = "Clear Sky";
+                                    } else if (skyType == SkyType::ClearTurbid) {
+                                        skyTypeString = "Clear Turbid Sky";
+                                    } else if (skyType == SkyType::Intermediate) {
+                                        skyTypeString = "Intermediate Sky";
+                                    } else if (skyType == SkyType::Overcast) {
+                                        skyTypeString = "Overcast Sky";
+                                    //} else {
+                                    //    // Should never happen
+                                    //    skyTypeString = "ERROR_SKY_TYPE_NOT_HANDLED";
+                                    }
+
+                                    for (int refPtNum = 1; refPtNum <= ZoneDaylight(ZoneNum).TotalDaylRefPoints; ++refPtNum) {
+                                        DaylFac = ZoneDaylight(ZoneNum).DaylIllFacSky(12, 1, static_cast<int>(skyType), refPtNum, loop);
+                                        print(outputFiles.eio,
+                                              " {},{},{},{},{},{:.4R}\n",
+                                              skyTypeString,
+                                              CurMnDy,
+                                              Zone(ZoneNum).Name,
+                                              Surface(IWin).Name,
+                                              DaylRefPt(ZoneDaylight(ZoneNum).DaylRefPtNum(refPtNum)).Name,
+                                              DaylFac);
+                                    }
+                                }
```

Comment:
EIO output change. Loop on SkyType is cosmetic to make code dryer. Loop on reference point is key.